alembic-autogenerate.py

```diff
@@ -43,6 +44,7 @@
 # Loop over each of the instruments
 pattern = os.environ["SDM_SCHEMAS_DIR"] + "/yml/cdb_*.yaml"
 instruments = [re.search(r"cdb_(.+)\.yaml$", file).group(1) for file in glob.glob(pattern)]
+instruments = []  # you can just hard code the few we want
```

Comment:
You do not want to force the person (or GitHub workflow, etc.) running the migration to manually modify this file in order to migrate specific schemas.

Rather, I'd recommend that this come from a config file, command line arguments, etc.

Reply:
Agreed, we talked about this and decided to hard code the instruments.